framework-reference/appsmith.md

```diff
@@ -63,6 +64,56 @@ This object contains the data of the currently authenticated user.
 }
 ```
 
+### Geolocation
+
+This object contains functions to request the current user location and also the coordinated recived from this request
```

Comment:
I think there are other places this needs to be references like here
https://docs.appsmith.com/core-concepts/writing-code/appsmith-framework